reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: single
+title:  "Resource Status"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+*If you're not sure what a Managed Resource is, check out our [overview of Managed Delivery](/reference/managed-delivery) first.*
+
+## Overview
+Managed Resources will always have a **status** which describes the current state of the resource from Spinnaker's perspective. A resource's status can help you quickly answer questions like:
+  - Is Spinnaker taking automatic action on this resource right now?
+  - Does the actual configuration of this resource match the declarative configuration I gave to Spinnaker?
+  - Is something wrong with this resource?
+
+## How to find a resource's status
+
+### UI
+On the Infrastructure views (Clusters, Load Balancers, Firewalls), you'll see a color-coded "M" logo attached to any resources that are managed by Spinnaker. Hovering over the logo reveals additional context about the current status and what it means.
+
+Example on a cluster:
+{%
+  include
+  figure
+  image_path="./cluster-resource-status-ui.png"
+%}
+
+
+### API
+If you're [setup to use the API](/reference/managed-delivery/getting-started/#env-setup) and have the [ID of a resource](/reference/managed-delivery/getting-started/#find-a-resources-id) you can use it to call the `/managed/resources/<resourceId>/status` endpoint.
+
+Example request:
+```bash
+curl -X GET --header "X-SPINNAKER-USER: ${SPINNAKER_USER}" "${SPIN_URL}/<resourceId>/status"
+```
+Example response:
+```json
+{ "status": "HAPPY" }
+```
+
+## Status Reference
+
+### Happy
+{%
+  include
+  figure
+  image_path="./resource-status-happy.png"
+  caption="Example of a resource with the Happy status"
+%}
+
+When a resource has a Happy status, all is well and the actual state of the resource matches its declarative configuration.
+
+If Spinnaker detects a drift from the declarative configuration in the future, the resource will change status to [Diff](/reference/managed-delivery/resource-status/#diff). Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
```

Comment:
This same paragraph appears repeatedly in most resource status descriptions. I’d suggest moving it up before the first status header and adding the word “actively” in “While Spinnaker is actively managing a resource” to address the cases when this does not apply.

Reply:
Here's my dilemma:

1. I see a status in Spinnaker via the UI, I'm new to this and a bit confused about the specifics
2. I click on the 'Learn more about this' link because I'm thinking 'I don't wanna mess with anything until I know what's up with this status, what if I break it'
3. I get deep-linked right to the current status I'm seeing and I read the description for it, maybe another couple that are cross-linked from the description too
4. Now I'm feeling a bit more confident. Seems like I understand what's going on and I'm ready to be a team player with the Spinnaker machine
5. I go back to the UI and make a manual change that seems reasonable to me **because I never scrolled up and read the beginning of the reference section**.
6. My manual change looks like it's working, but then it doesn't.  Now I'm confused and it seems like this thing is broken.

Am I being too cautious? I ask that question genuinely because I'm not sure and I want to do the right thing. My greatest fear is that the feedback and context from this UI -> docs workflow _fails to prevent an avoidable incident_.

Reply:
I agree with you @erikmunson. Let's repeat things in every section if it's meant to be a reference (exact same wording every time == "oh I've read this before I understand and I'm comfortable with the meaning"). These are new and scary concepts and will take people at least 3x of reading something to actually learn it.

Reply:
Agreed as well! I didn’t think about how users would land in each section when I made that comment.